nav2_graceful_controller/src/graceful_controller.cpp

```
@@ -199,24 +155,109 @@ geometry_msgs::msg::TwistStamped GracefulController::computeVelocityCommands(
       logger_, "Could not transform %s to %s: %s",
       costmap_ros_->getBaseFrameID().c_str(), costmap_ros_->getGlobalFrameID().c_str(),
       ex.what());
-    return cmd_vel;
+    throw nav2_core::NoValidControl("Could not transform");
```

You can just re-throw the TF exception `throw ex;`